packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -16,11 +16,12 @@ import { plugin as requestAuthPlugin } from 'mwp-auth-plugin';
  *
  * @module ServerPlugins
  */
-
-const CSRF_COOKIE_NAME = 'x-mwp-csrf';
+const CSRF_COOKIE_NAME =
+	process.env.NODE_ENV === 'production' ? 'x-mwp-csrf' : 'x-mwp-csrf_dev';
```

Comment:
@mmcgahan you suggested `_DEV` (in caps) but it compares the cookie + header in all lowercase so when this is _DEV it will always fail the csrf token comparison. I didn't dig too deeply to figure out exactly where it's changing to lowercase, but figured having `_DEV` in caps for the cookie/header name isn't a dealbreaker

Reply:
ah ok yeah I think Hapi converts all header names to lowercase, so that makes sense!